src/humanloop/otel/__init__.py

```diff
@@ -0,0 +1,80 @@
+from opentelemetry import baggage
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.trace import Tracer
+
+from humanloop.otel.constants import HL_TRACE_METADATA_KEY
+from humanloop.otel.helpers import module_is_installed
+
+
+_TRACER = None
+
+_BAGGAGE_CONTEXT = [{}]
+
+
+def set_tracer(tracer: Tracer):
+    global _TRACER
+    _TRACER = tracer
+
+
+def get_tracer() -> Tracer:
+    assert _TRACER is not None, "Internal error: OTT Tracer should have been set in the client"
+    return _TRACER
+
+
+def instrument_provider(provider: TracerProvider):
+    """Add Instrumentors to the TracerProvider.
+
+    Instrumentors add extra spans which are merged in Humanloop Span logs.
+    """
+    if module_is_installed("openai"):
```

Comment:
This feels brittle. 

Does it deal with different versions of the package ok? Are there scenarios where module_is_installed won't find the module?

Reply:
Should be fine - it checks if the module is importable, no matter the version.

With the LLM providers included in the current PR, Instrumentors have no requirements on the version of the client library that is intercepted

Reply:
> Are there scenarios where module_is_installed won't find the module?

Could you comment on this?

Reply:
No - this is a programmatic way to simulate the developer importing the module. If they have it installed in their pyenv, the helper will find it.